variables.tf

```diff
@@ -105,6 +105,15 @@ variable "api_mapping_key" {
   default     = null
 }
 
+variable "additional_default_stage_api_mappings" {
```

Comment:
lets change this to additional_stage_api_mapping because we can name created stage now